src/doc/rustc/src/command-line-arguments.md

```diff
@@ -62,11 +62,15 @@ attribute or on the command-line, it will link a dynamic library if available,
 otherwise it will use a static library. If the kind is specified on the
 command-line, it will override the kind specified in a `link` attribute.
 
+If you don't specify `KIND`, no linkage modifiers will be applied.
+For instance, [`+bundle` linkage modifier][bundle-modifier] won't be used in conjunction with static library by default.
```

Comment:
Of course it won't be used, because there won't be any static library.
When this PR was submitted, unspecified kind always meant `dylib`.